gst/nnstreamer/tensor_aggregator/tensor_aggregator.c

```diff
@@ -292,7 +292,12 @@ gst_tensor_aggregator_init (GstTensorAggregator * self)
   self->frames_dim = DEFAULT_FRAMES_DIMENSION;
   self->concat = DEFAULT_CONCAT;
 
+  self->tensor_configured = FALSE;
+  gst_tensor_config_init (&self->in_config);
+  gst_tensor_config_init (&self->out_config);
+
   self->adapter = gst_adapter_new ();
+  g_assert (self->adapter != NULL);
```

Comment:
Linux 295-297 is executed by ```gst_tensor_aggregator_reste()``` at Line 301. Why duplicate?

Reply:
The tensor config init part in `gst_tensor_aggregator_reset` is moved to L295-297.